BL-5109 Pan/Zoom animation

JohnThomson opened the pull request:
Requires BloomPlayer changes, which should be merged first

<!-- Reviewable:start -->
---
This change is [<img src="https://reviewable.io/review_button.svg" height="34" align="absmiddle" alt="Reviewable"/>](https://reviewable.io/reviews/bloombooks/bloomreader/64)
<!-- Reviewable:end -->

andrew-polk commented:
Reviewed 3 of 3 files at r1.
Review status: all files reviewed at latest revision, 2 unresolved discussions.

---

*[app/src/main/java/org/sil/bloom/reader/ReaderActivity.java, line 207 at r1](https://reviewable.io:443/reviews/bloombooks/bloomreader/64#-Kxwi3LFDcZjHx7g1GhY:-Kxwi3LFDcZjHx7g1GhZ:b-xv7aoa) ([raw file](https://github.com/bloombooks/bloomreader/blob/2dfc927a7bd5492f1b5c4d74e72b5dc0712fc0cf/app/src/main/java/org/sil/bloom/reader/ReaderActivity.java#L207)):*
> ```Java
>             if(mSwitchedPagesWhilePaused) {
>                 final int position = mPager.getCurrentItem();
> \                mAdapter.startNarrationForPage(position); // also starts animation if any
> ```

This line starts with a "\"
Probably a miskey?

---

*[app/src/main/java/org/sil/bloom/reader/WebAppInterface.java, line 71 at r1](https://reviewable.io:443/reviews/bloombooks/bloomreader/64#-KxwikaB9zrElbAT8u0O:-KxwikaB9zrElbAT8u0P:bots7hb) ([raw file](https://github.com/bloombooks/bloomreader/blob/2dfc927a7bd5492f1b5c4d74e72b5dc0712fc0cf/app/src/main/java/org/sil/bloom/reader/WebAppInterface.java#L71)):*
> ```Java
>     }
> 
>     public void playPause(boolean pause) {
> ```

This playPause method confused me when I saw it in consuming code.
Should we split it into two methods, play and pause?

---


*Comments from [Reviewable](https://reviewable.io:443/reviews/bloombooks/bloomreader/64)*
<!-- Sent from Reviewable.io -->

JohnThomson commented:
Review status: 1 of 3 files reviewed at latest revision, 2 unresolved discussions.

---

*[app/src/main/java/org/sil/bloom/reader/ReaderActivity.java, line 207 at r1](https://reviewable.io:443/reviews/bloombooks/bloomreader/64#-Kxwi3LFDcZjHx7g1GhY:-Kxx-PqBJTQrW8alFzlj:b-896fix) ([raw file](https://github.com/bloombooks/bloomreader/blob/2dfc927a7bd5492f1b5c4d74e72b5dc0712fc0cf/app/src/main/java/org/sil/bloom/reader/ReaderActivity.java#L207)):*
<details><summary><i>Previously, andrew-polk wrote…</i></summary><blockquote>

This line starts with a "\"
Probably a miskey?
</blockquote></details>

Done.

---

*[app/src/main/java/org/sil/bloom/reader/WebAppInterface.java, line 71 at r1](https://reviewable.io:443/reviews/bloombooks/bloomreader/64#-KxwikaB9zrElbAT8u0O:-Kxx-xT_p6n2YOgOYnBg:b-5q2hd9) ([raw file](https://github.com/bloombooks/bloomreader/blob/2dfc927a7bd5492f1b5c4d74e72b5dc0712fc0cf/app/src/main/java/org/sil/bloom/reader/WebAppInterface.java#L71)):*
<details><summary><i>Previously, andrew-polk wrote…</i></summary><blockquote>

This playPause method confused me when I saw it in consuming code.
Should we split it into two methods, play and pause?
</blockquote></details>

I changed it to setPaused(). Two methods mean all (admittedly, only 2) clients have to duplicate logic like if (paused) play() else pause()

---


*Comments from [Reviewable](https://reviewable.io:443/reviews/bloombooks/bloomreader/64)*
<!-- Sent from Reviewable.io -->

andrew-polk commented:
<img class="emoji" title=":lgtm:" alt=":lgtm:" align="absmiddle" src="https://reviewable.io/lgtm.png" height="20" width="61"/>

---

Reviewed 2 of 2 files at r2.
Review status: all files reviewed at latest revision, all discussions resolved.

---



*Comments from [Reviewable](https://reviewable.io:443/reviews/bloombooks/bloomreader/64#-:-Kxx0ybU-CXYCNgY-HX2:bnfp4nl)*
<!-- Sent from Reviewable.io -->
